src/docdb/getDocumentClient.ts

```diff
@@ -3,27 +3,18 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { DocumentClient } from "documentdb";
-import * as DocDBLib from 'documentdb/lib';
-import * as vscode from "vscode";
+import { CosmosClient } from "@azure/cosmos";
 import { appendExtensionUserAgent } from "vscode-azureextensionui";
-import { ext } from "../extensionVariables";
 
-export function getDocumentClient(documentEndpoint: string, masterKey: string, isEmulator: boolean | undefined): DocumentClient {
-    const documentBase = DocDBLib.DocumentBase;
-    const connectionPolicy = new documentBase.ConnectionPolicy();
-
-    const vscodeStrictSSL: boolean | undefined = vscode.workspace.getConfiguration().get<boolean>(ext.settingsKeys.vsCode.proxyStrictSSL);
-    const strictSSL = !isEmulator && vscodeStrictSSL;
-    connectionPolicy.DisableSSLVerification = !strictSSL;
-    const client = new DocumentClient(documentEndpoint, { masterKey: masterKey }, connectionPolicy);
+export function getDocumentClient(endpoint: string, key: string): CosmosClient {
+    const client = new CosmosClient({ endpoint, key });
 
     // User agent isn't formally exposed on the client (https://github.com/Azure/azure-documentdb-node/issues/244) but nevertheless can be accessed via defaultHeaders
```

Comment:
Is there a better way to do this now that we're on a new sdk?

Reply:
No, `UserAgentSuffix` is not exposed on the client. The only change is that with the new SDK we don't to access the suffix through `defaultHeaders`.